### ISSUE TITLE
can't open the link

### ISSUE THREAD

**Issue by coldrainy** [udacity/sdc-issue-reports#1414]
In the term2 lesson 18 vehicle model.I can't open the link of the optional content which is:
https://classroom.udacity.com/nanodegrees/nd013/parts/40f38239-66b6-46ec-ae68-03afd8a601c8/modules/0949fca6-b379-42af-a919-ee50aa304e6a/lessons/4c4f9180-7ab8-42ba-a1bd-24b17fcf2178/concepts/3d7a14af-6afa-446e-9399-622360eddd6c

**Comment by mvirgo** [comment]
Good catch - I believe the link slightly changed when we moved it out of the main section of the Nanodegree and into Extracurriculars. I've updated the link in the classroom, can you let me know if it works for you now? (It might still get cached with the old link)

**Comment by Anner-deJong** [comment]
Same here. (Working in chrome on a macbook, although I think that's irrelevant)
I get redirected to [_home_](https://classroom.udacity.com/me) (https://classroom.udacity.com/me)
